.gitignore

```diff
@@ -51,6 +51,7 @@ install/kubernetes/istio-one-namespace.yaml
 install/kubernetes/istio-sidecar-injector-configmap-debug.yaml
 install/kubernetes/istio-sidecar-injector-configmap-release.yaml
 install/kubernetes/istio-sidecar-injector.yaml
+install/kubernetes/istio-auth-sidecar-injector.yaml
```

Comment:
Please stop adding more files here - we need to get rid of the existing ones instead.

Run helm template with the right parameters, generate a file in the $OUT directory - and change the test to read from there.

No 'updateVersion.sh' either - just helm template (in a make target)